### ISSUE TITLE
Doesn't work on local pages

### ISSUE THREAD

**Issue by DuncanCasteleyn** [h5o/h5o-chrome#15]
This outliner tool can't be used on local html pages even though there is extension option in chrome that allows extensions to access local file urls. The webdevelopper tool has this option and it allows you to validate the local content of the pages.

**Comment by dominykas** [comment]
I did not enable `file://`permissions when I first built this and I am reluctant to add it now to the main extension that lives in Chrome Store, however you can install it from here: https://github.com/h5o/h5o-chrome/releases and it should then work.

Duplicate of #4.

**Comment by marctorsoc** [comment]
Labas @dominykas (yeah, I speak some Lithuanian), this extension is amazing and I've been using it for years. Thanks a lot for developing. However, now I'm developing html locally, and I don't get how to "install from here" from your previous comment.

I downloaded the zip and got the source code from [here](https://github.com/h5o/h5o-chrome/releases). Then followed, [this](https://dev.to/ben/how-to-install-chrome-extensions-manually-from-github-1612), but I get 
```
Required value 'version' is missing or invalid. It must be between 1-4 dot-separated integers each between 0 and 65536.
Could not load manifest.
```
I tried with both 0.8.14 and 0.8.13. 

Any hints would be much appreciated
Ačiū

**Comment by marctorsoc** [comment]
FWIW, I tried adding 
```
"version": "0.8.14", 
```
as suggested [here](https://stackoverflow.com/questions/32928473/chrome-required-value-version-is-missing-or-invalid-it-must-be-between-1-4-do), and now getting
```
Could not load JavaScript 'outliner.min.js' for script.
Could not load manifest.
```

**Comment by marctorsoc** [comment]
ok so for anyone who comes here, I got how to install. One has to follow the steps in the README (I modified them a bit)
```
npm install grunt-cli
grunt
```
This will create `dist/extension` and uploading `extension` to `Load unpacked` works.

However, the extension I got does exactly the same as the one in the Chrome store. I tried with 0.8.14, 0.8.13 and 0.8.3 (the oldest available). I wonder if any version really supports this...
